### PR TITLE
Improve documentation about package sync between two pulp servers

### PR DESCRIPTION
**Pull request by mikeadamz**
This is a re-submit of PR #112 with the requested changes made.

fixes 2900

### CODE REVIEW

File: docs/user-docs/getting_started.rst [hunk 1]
```diff
@@ -185,3 +185,11 @@ used as a feed for a second Pulp. Here is an example.::
    $ pulp-admin python repo create --repo-id sync_from_other_pulp --feed http://<host>/pulp/python/web/<repo_id>/ --package-names numpy,scipy
 
    $ pulp-admin python repo sync run --repo-id sync_from_other_pulp
+
+The pulp_python RPM distribution automatically adds index.html and index.json as a DirectoryIndex
+in pulp_python.conf. If you're not running the pulp_python RPM distribution, or you have
+customized your apache configuration, you will need to ensure this setting is present before remote
+Pulp instances will be able to sync your repository.
```
Comment (asmacdo) [inline docs/user-docs/getting_started.rst]: @mikeadamz This is now a dupe, would you mind removing this?

**Comment by asmacdo** [comment]
If you aren't comfortable squashing, these docs should help:
http://docs.pulpproject.org/dev-guide/contributing/merging.html#rebasing-and-squashing

**Comment by asmacdo** [comment]
https://pulp.plan.io/issues/2900

**Comment by mikeadamz** [comment]
I'll get this done. I've just been out of pocket the last few days; seeing the oral surgeon today. I should be back up and around tomorrow or thursday.

**Comment by mikeadamz** [comment]
Take a look now, it looks right to me but I'm not 100% sure.

**Comment by asmacdo** [comment]
@mikeadamz, I've manually associated the commit to the issue, so we are all set.  FYI, it didn't happen automatically because there was a subtle commit message syntax error, it should have been `fixes #2900` not `fixes 2900`.

**Comment by mikeadamz** [comment]
ah okay, thanks! I'll make sure its right next time.

**Pull request merged**